python/run_simulation.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+"""
+Running the individual-based model, COVID19-IBM
+
+Usage:
+python3 run_simulation.py  
+
+Created: May 2020
+Author: Dylan Feldner-Busztin
+"""
+
+import subprocess, pytest, os, sys
+import numpy as np, pandas as pd
+from . import constant
+
```

Comment:
Please remove all the comments if the data just comes from `constant`

Reply:
Removed.

Reply:
Removed comments.